### DISCUSSION TITLE
Nothing happens after pressing `Continue` in the `Application` box.

### DISCUSSION THREAD

**Discussion by TornaxO7** [community/community#139185] [GitHub Education] [answered]
### Hi everyone,

I'm currently trying to re-apply for the github education program.
However, after pressing the `Continue` button nothing happens.

It looks like this:
![image](https://github.com/user-attachments/assets/1ac22a3d-36a6-4a5b-bcf7-4a6e145aa792)

and it stays like this.
I've enabled location sharing. I've opened this from firefox (output of `firefox --verison`: Mozilla Firefox 130.0). Does anyone has an idea how to fix this?

**Comment by davevad93** [comment, marked as answer]
Hi @TornaxO7 ,

This issue has been addressed in many discussions in the past and most users solved it by doing one of the following:

1. Clearing the browser cache.
2. Using a different browser.
3. Disabling any browser extensions that may be interfering with `JavaScript`.
5. Submitting the application with the smartphone.
